clients/drcachesim/tests/allasm_repstr.asm

```diff
@@ -76,3 +83,6 @@ hello_str:
         .string  "Hello world!\n"
 bye_str:
         .string  "Adios\n"
+        .align   4096
```

Comment:
Add comment that we need this for the multi-page check.

Reply:
Done, along w/ assumption on 4K pages.